src/lib/moment/calendar.js

```diff
@@ -14,6 +15,12 @@ export function getCalendarFormat(myMoment, now) {
 }
 
 export function calendar (time, formats) {
+    // #3658 - Adding overload to the calendar function in order to allow:
```

Comment:
You can say in a comment above the `if` along the line of `accept format only`, but don't include the PR/issue number, there is git blame for that.